ext/lib/mgmt/mcumgr/samples/smp_svr/zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# Top-level CMakeLists.txt for the skeleton application.
+#
+# Copyright (c) 2017 Open Source Foundries Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This provides a basic application structure suitable for communication using
+# mcumgr.  It can be used as a starting point for new applications.
+
+# The default top-level application configuration is prj.conf.  You can place
+# additional board-specific files in boards/${BOARD}.conf, and they will be
+# merged into the configuration along with prj.conf.
+macro(set_conf_file)
+  if(EXISTS ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf)
+    set(CONF_FILE "prj.conf ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf")
+  else()
+    set(CONF_FILE "prj.conf")
+  endif()
+endmacro()
```

Comment:
This macro and it's associated documentation should be removed. This boilerplate is non-standard for samples and there is nothing specific about mcumgr that warrants it.

Reply:
That must be a leftover from the original `CMakeLists.txt` file I used as a template (from MCUboot, I believe).  Thanks for catching this; I have removed it.